ee-security/README.adoc

```diff
@@ -59,9 +64,9 @@ process-state: reload-required
 ----
```

Comment:
The Readme instructions doesn't contain `reload` command after the `.cli` script is executed. If server isn't reloaded, the tests and manual check via `curl` would fail.